src/octoprint/plugins/classicwebcam/static/js/classicwebcam_settings.js

```diff
@@ -0,0 +1,212 @@
+$(function () {
```

Comment:
Am I right in assuming that this is moved from `src/static/js/app/viewmodels/settings.js`?

Reply:
Yes, but with some changes to accommodate the new setup, e.g.the settings paths changes to the plugin's settings